autogpt/commands.py

```diff
@@ -175,9 +176,9 @@ def browse_website(url, question):
     return result
 
 
-def get_text_summary(url, question):
+def get_text_summary(url):
```

Comment:
Another feature this PR removes, unrelated to the topic of change.

Reply:
This is exactly why we want PRs to be focused. It's easy to overlook.